### PR TITLE
Feature/auto charmhub publish

### PR DESCRIPTION
**Pull request by paulomach**
## Proposal

Auto publish to charmhub latest/edge github workflow.
Charmhub access key added in repository secrets.

### CODE REVIEW

**Review by jnsgruk — DISMISSED** [review]

**Review by WRFitch — CHANGES_REQUESTED** [review]
Pipeline looks great, thanks for submitting this Paulo! 

I'm currently not comfortable with releasing the charm even on /edge, since it currently doesn't work. Once PRs #10 and #13 are merged, the relations should be fully functional, and I'll be happy to merge this

**Review by delgod — APPROVED** [review]

**Comment by WRFitch** [comment]
Update on this PR, we're now waiting on PRs #18 and #19, and then I'm happy to sort out merge conflicts and get this merged in :)

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      - name: Run integration tests
+        run: tox -e integration
+
+  release-to-charmhub:
+    name: Release to CharmHub
+    needs:
+      - lib-check
+      - lint
+      - unit-test
+      - integration-test-lxd
```
Comment (marceloneppel) [inline .github/workflows/release.yaml:64]: Looks good to be merged after the other two PRs that Will commented. Just a reminder: before merging this PR, this line should be changed (and also some jobs need to be added):
```suggestion
      - integration-standalone
      - integration-backend
      - integration-legacy-relations
```

**Review by marceloneppel — APPROVED** [review]
Looks great!

**Review by WRFitch — APPROVED** [review]

**Pull request merged**